src/Datadog.Trace.ClrProfiler.Managed.Loader/StartupLogger.cs

```diff
@@ -0,0 +1,150 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+
+namespace Datadog.Trace.ClrProfiler.Managed.Loader
+{
+    internal static class StartupLogger
+    {
+        private const string NixDefaultDirectory = "/var/log/datadog/dotnet";
+
+        private static readonly string LogDirectory = GetLogDirectory();
+        private static readonly string StartupLogFilePath = SetStartupLogFilePath();
+
+        /// <summary>
+        /// Gets a value indicating whether this OS is Windows.
+        /// Prevents the need for a direct System.Runtime reference.
+        /// </summary>
+        public static bool IsWindows
+        {
+            get
+            {
+                var p = (int)Environment.OSVersion.Platform;
```

Comment:
Why cast to `int`? Also, the only value we need to check for is `System.PlatformID.Win32NT` (2).